dev_scripts/get_apk_versions

```diff
@@ -10,22 +10,31 @@ sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
 
 from wlauto.exceptions import WAError, ToolError
 from wlauto.utils.doc import format_simple_table
+from wlauto.utils.misc import (check_output, get_null)
 
 
 def get_aapt_path():
     """Return the full path to aapt tool."""
     sdk_path = os.getenv('ANDROID_HOME')
-    if not sdk_path:
-        raise ToolError('Please make sure you have Android SDK installed and have ANDROID_HOME set.')
-    build_tools_directory = os.path.join(sdk_path, 'build-tools')
-    versions = os.listdir(build_tools_directory)
-    for version in reversed(sorted(versions)):
-        aapt_path = os.path.join(build_tools_directory, version, 'aapt')
-        if os.path.isfile(aapt_path):
-            logging.debug('Found aapt for version {}'.format(version))
-            return aapt_path
+    if sdk_path:
+        build_tools_directory = os.path.join(sdk_path, 'build-tools')
+        versions = os.listdir(build_tools_directory)
+        for version in reversed(sorted(versions)):
+            aapt_path = os.path.join(build_tools_directory, version, 'aapt')
+            if os.path.isfile(aapt_path):
+                logging.debug('Found aapt for version {}'.format(version))
+                return aapt_path
+        else:
+            raise ToolError('aapt not found. Please make sure at least one Android platform is installed.')
     else:
-        raise ToolError('aapt not found. Please make sure at least one Android platform is installed.')
+        logging.debug("ANDROID_HOME is not set, try to find in $PATH.")
+        if subprocess.call('aapt version >{} 2>&1'.format(get_null()), shell=True) == 0:
+            aapt_path = os.path.dirname(subprocess.check_output('which aapt', shell=True))
+            aapt_path = os.path.join(aapt_path, 'aapt')
+            logging.debug('Using aapt from {}'.format(aapt_path))
+            return aapt_path
```

Comment:
You should use 'which()' here, rather than using OS ultils directly:

https://github.com/ARM-software/workload-automation/blob/master/wlauto/utils/misc.py#L605

This will also eliminate the need to call `aapt version` beforehand. This if statement can be replaced with
```
aapt_path = which('aapt')
if aapt_path:
    return aapt_path
raise ToolError('Please make sure you have Android SDK installed and have ANDROID_HOME set.')
```

Reply:
Oh!!! Thank you for pointing it out, I did not notice that there is a useful wrapper!!